boms/cloud-lts-bom/RELEASING.md

```diff
@@ -11,16 +11,39 @@ All on your corp desktop:
 
 1. Run gcert if you have not done so in the last twelve hours or so.
 
-2. Run `release.sh` with `lts` argument in 
+2. If this is a major version release, create a LTS release branch `N.0.x-lts`, where
```

Comment:
Now this operation includes creating an LTS release branch before performing a release.